src/compas_view2/app/controller.py

```diff
@@ -33,11 +33,77 @@ def __init__(self, app, controller_config: Dict):
         self.keys = controller_config["actions"]["keys"]
         self.mouse = Mouse()
         self.key_status = {}
-        self.actions = {}  # Currently, no customized actions supported.
+        self.actions = self.keys.keys()
         self.supported_keys = {
```

Comment:
This variable ` self.supported_keys` is a bit long, but should be fine for this version